app/models/user.rb

```diff
@@ -1,8 +1,12 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, 
+         :recoverable, :rememberable, :trackable, :validatable
   include SnoopDogg
 
   rolify
-  devise :database_authenticatable, :registerable,
+  devise :database_authenticatable, 
```

Comment:
@PaulStorberg Put these one one line. They are not many of them. They fit on one line.

Reply:
complete